ts/features/mvl/index.ts

```diff
@@ -40,7 +47,6 @@ export const mvlPersistConfig: PersistConfig = {
   key: "mvl",
   storage: AsyncStorage,
   version: CURRENT_REDUX_MVL_STORE_VERSION,
-  whitelist: ["preferences"],
```

Comment:
Removing the `whilelist` configuration should lead to accidentally starting persisting the `byId` configuration (which was not saved before).

At this point, I think that we are not longer in need of persisting anything at all from MVL so what if we:
- "downgrade" the MVL from a persisted reducer to an in-memory only reducer
- add a new migration into the root persist configuration, where we just remove the `persist:mvl` key from the AsyncStorage?

Reply:
Ops, nice catch! Yes, your solution is fine

Reply:
Another alternative is to set an empty whitelist, in this way we don't persist `byId`.
What do you think about it?

Reply:
Doing so will result in a persistence configuration that persists nothing. However, given the scope of this PR, it can be acceptable. Up to you